tests/integration/fixtures/cubes/scale_means/cat-x-cat-sm-margin.json

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
This looks like a multitable export, but I'm not sure it should be. It's not a _big deal_ but a CrunchCube response alone should be sufficient here and be a little bit less distracting as to what's going on.

Reply:
It's a deck export. I can make it a pure cube JSON.

Reply:
Changed to explicit cube.